src/software/ai/hl/stp/tactic/receiver/receiver_tactic.h

```diff
@@ -12,17 +12,27 @@
  * Note that this tactic does not take into account the time the pass should occur at,
  * it simply tries to move to the best position to receive the pass as possible
  */
-class ReceiverTactic : public Tactic
+class ReceiverTactic : public SupportTactic
 {
    public:
     explicit ReceiverTactic(std::shared_ptr<Strategy> strategy);
 
     void accept(TacticVisitor& visitor) const override;
 
-    std::map<RobotId, std::shared_ptr<Primitive>> get(const WorldPtr &world_ptr) override;
-
+    void updateReceivingPosition(std::optional<Point> receiving_position) override;
+    
     DEFINE_TACTIC_DONE_AND_GET_FSM_STATE
 
+    /**
+     * Updates the control parameters for this ReceiverTactic.
+     *
+     * @param receiving_position The point at which to receive the pass
+     * @param disable_one_touch_shot If set to true, the receiver will not perform a
+     * one-touch The robot will simply receive and dribble.
+     */
+    void updateControlParams(std::optional<Point> receiving_position,
+                             bool disable_one_touch_shot = false);
```

Comment:
nit: `enable_one_touch_shot` is nicer to understand logically because you don't have to work with double negatives